tests/test_minhash.py

```diff
@@ -1872,6 +1887,27 @@ def test_merge_scaled():
     for k in mh2.hashes:
         assert k in mh3.hashes
 
+def test_add_is_symmetric():
+    mh1 = MinHash(20, 21)
+    mh1.add_hash(5)  
+    mh2 = MinHash(20, 21)
+    mh2.add_hash(6)
+    print("\n mh1 EQUALS ", mh1.hashes, "\n mh2 EQUALS", mh2.hashes)
+    mh3 = mh1 + mh2
+    mh4 = mh2 + mh1
+    print("\n mh3 EQUALS ", mh3.hashes, "\n mh4 EQUALS", mh4.hashes)
+    #if mh3 != 0, then it is "true", so it passes
+    assert mh3
+    assert mh3 == mh4
+
+
+def test_or_equals_add():
```

Comment:
This test still uses empty minhashes, so doesn't properly test or equal addition.

Reply:
wait how are they still empty? isnt mh1={5:1} and mh2={6:1}, so mh3=mh4={5:1,6:1}?

Reply:
this test function, `test_or_equals_add`, creates new mh1 and mh2 objects that are empty - or at least the version I see below does, right?

Reply:
(each `def test_` function block is a namespace independent of all of the others)

Reply:
OH ok, wrong line number. I was looking at the test above. my bad, thx for catching